src/gov-action-contracts/address-registries/L1AddressRegistry.sol

```diff
@@ -32,4 +38,12 @@ contract L1AddressRegistry is IL1AddressRegistry {
     function sequencerInbox() public view returns (ISequencerInbox) {
         return inbox.sequencerInbox();
     }
+
+    function getOutboxes() public view returns (address[] memory) {
+        return outboxes;
+    }
+
+    function getSequencers() public view returns (address[] memory) {
+        return sequencers;
```

Comment:
i think sequencers can be expected to change, likewise outboxes, so we probably dont want to hardcode them here. I think it would be better to add some pause functionality to the bridge instead.

Reply:
hmm yeah agreed; pausable bridge let's us more directly both pause sequencing, outbox execution, and gives means to pause force inclusion, v nice. Kinda like the idea of pushing forward with that in the short/ASAP - term. (tho in the immediate term maybe we temporarily have this? idk)